Marlin/src/HAL/STM32F1/MarlinSerial.cpp

```diff
@@ -28,7 +28,11 @@
 
 // Copied from ~/.platformio/packages/framework-arduinoststm32-maple/STM32F1/system/libmaple/usart_private.h
 // Changed to handle Emergency Parser
-static __always_inline void my_usart_irq(ring_buffer *rb, ring_buffer *wb, usart_reg_map *regs, MSerialT &serial) {
+static
+#if MB(VOXELAB_AQUILA)
```

Comment:
Instead of directly checking the MB here, define a flag (e.g., "`STATIC_INLINE_USART_IRQ`") in its pins file. It will then be included by `MarlinConfig.h`. `_TERN(STATIC_INLINE_USART_IRQ, inline)`.

Or, is it that `__always_inline` is defined in such a way that the extra `inline` is required everywhere for a certain STM32F1? If that is the case `__always_inline` could be patched in one of the `HAL/STM32F1/inc/Conditionals_*.h` files.